src/Microsoft.AspNetCore.Server.IIS/HttpProtocol.cs

```diff
@@ -96,6 +98,7 @@ internal unsafe HttpProtocol(PipeFactory pipeFactory, IntPtr pHttpContext)
 
                 // TODO: Avoid using long.ToString, it's pretty slow
                 RequestConnectionId = ConnectionId.ToString(CultureInfo.InvariantCulture);
+                UConnectionId = ConnectionId;
```

Comment:
What is UConnectionId?

Reply:
It was used by HttpSys for RequestAborted, but I'll probably take the logic from Kestrel here.

Reply:
No need to copy from kestrel, I'm just trying to understand why we have 2 properties:
 UConnectionId and ConnectionId.

Reply:
I'll drop it for now until I implement the feature.